src/Api/Request/Request.php

```diff
@@ -143,7 +143,13 @@ public function setAttributeParam($key, $value)
             $this->params[Plugin::API_PARAMETER_ATTRIBUTES] = [];
         }
 
-        $this->params[Plugin::API_PARAMETER_ATTRIBUTES][$key] = $value;
+        // Remove duplicate entries, since the plenty request may give them as duplicates. Additionally
+        // check that min/max is not set, because the range-slider may have the same min and max value.
+        if (is_array($value) && (!isset($value['min']) && !isset($value['max']))) {
```

Comment:
With this logic we don't remove duplicates when the price slider is used, but what happens when several filters are in use and one of them is the price slider?
I guess a solution to exclude any min and max values from the `array_unique` would be more specific.

Reply:
This is already per-filter. Basically it is like first vendor then price, etc. So we are only getting the unique values per-filter already. Added another test to reflect that.